FAQUse: Misc update or remove outdated information

turbolay opened the pull request:
Explanation of changes:

1) Even if I was also a frequent user of https://jochen-hoenicke.de/queue/#0,24h, nowadays https://mempool.space/ UI has more info and is easier to understand
2) `Build Transaction` feature doesn't exist anymore in WW2 (except from RPC). This section is outdated.
3) `InputRegistration` in WW2 is different: 150 inputs min and fixed time frame. I didn't mention that this time frame is 10 minutes because it can change etc etc. If you want me to add please notify I will.
4) Same as 3)
5) https://testnet-faucet.mempool.co/ is dead
6) `if there are a lot of users` this doesn't apply to WW2
7) `How long does it take to mix my coins?` this section was written for WW1, it's completely irrelevant for WW2. We could rewrite it instead of a straight deletion, but other sections below like `How long does it take to make my wallet 100% private?` are already bringing updated info.
8) Non private coins are now grey not red anymore
9) `How can I mix large amounts?` this section was written for WW1, it's completely irrelevant for WW2. We could rewrite it instead of a straight deletion, but other sections are already bringing updated info

MarnixCroes reviewed (approved):
LGTM

yahiheb commented on docs/FAQ/FAQ-UseWasabi.md:
```diff
@@ -1271,7 +1249,7 @@ To avoid any privacy leak, you can use a Ledger hardware wallet in combination w
 
 :::details
-### After I CoinJoined my coins and reached green anonset, I sent them to my hardware wallet address. When I check my HW via Wasabi, the coins are now red. Why?
+### After I CoinJoined my coins and reached green anonset, I sent them to my hardware wallet address. When I check my HW via Wasabi, the coins are now grey. Why?
```
This breaks external links, and the content is still not up to date, let's not edited for now.

```suggestion
### After I CoinJoined my coins and reached green anonset, I sent them to my hardware wallet address. When I check my HW via Wasabi, the coins are now red. Why?
```

yahiheb commented on docs/FAQ/FAQ-UseWasabi.md:
```diff
@@ -1497,14 +1475,6 @@ For more information, see this [dedicated chapter](/using-wasabi/ChangeCoins.md)
 :::
 ::::
 
```
```suggestion
```

turbolay commented:
implemented all of your suggestions @yahiheb.
For the faucet, I found another dead link in the doc so I updated also, and I added https://bitcoinfaucet.uo1.net/ below befcause it sends much less coins than the other one.

yahiheb commented on docs/FAQ/FAQ-Installation.md:
```diff
@@ -195,7 +195,11 @@ Clicking on `Update` uses the default browser to go to the website, where you ca
 All releases (major and minor) are announced on the social media like [Twitter](https://twitter.com/wasabiwallet) and [Reddit](https://reddit.com/r/WasabiWallet/).
+
+:::tip Since Wasabi v2.0.1.4, an automatic downloader has been implemented
+Clients since v2.0.1.4 can simply turn on the setting `Auto download new version`, and all the work will be done in the background. More information is available in the dedicated [section](/FAQ/FAQ-Installation.md#how-does-the-auto-download-new-version)
```
Usually we write every sentence in a new line. https://github.com/zkSNACKs/WasabiDoc#writing-conventions

```suggestion
Clients since v2.0.1.4 can simply turn on the setting `Auto download new version`, and all the work will be done in the background.
More information is available in the dedicated [section](/FAQ/FAQ-Installation.md#how-does-the-auto-download-new-version).
```

yahiheb commented on docs/FAQ/FAQ-Installation.md:
```diff
@@ -211,19 +215,19 @@ For step-by-step instructions, follow [this guide](/using-wasabi/InstallPackage.
 If enabled, the software will automatically download the new version's installer upon a new release.
 If disabled, the software will only signal, but not automatically download, that there is a new version available.
 
```
```suggestion
```

yahiheb commented on docs/FAQ/FAQ-UseWasabi.md:
```diff
@@ -924,8 +915,6 @@ Go to `Settings` > `Bitcoin` and change the network to `TestNet`.
 You can get tBTC from faucets like:
-[testnet-faucet.mempool.co](https://testnet-faucet.mempool.co/)
-or
 [coinfaucet.eu/en/btc-testnet](https://coinfaucet.eu/en/btc-testnet/)
```
```suggestion
[coinfaucet.eu/en/btc-testnet](https://coinfaucet.eu/en/btc-testnet/) or [bitcoinfaucet.uo1.net](https://bitcoinfaucet.uo1.net/)
```

yahiheb commented on docs/using-wasabi/Testnet.md:
```diff
@@ -60,7 +60,7 @@ You can [generate a receive address](/using-wasabi/Receive.md) the same way as o
 
 Because testnet bitcoin don't have economical value, they are gifted by different sources.
-There are [faucets](https://en.bitcoin.it/wiki/Testnet#Faucets) like [this one](https://testnet-faucet.mempool.co/) that give a certain amount of testnet bitcoin per time period.
+There are [faucets](https://en.bitcoin.it/wiki/Testnet#Faucets) like [this one](https://coinfaucet.eu/en/btc-testnet/) that give a certain amount of testnet bitcoin per time period.
```
```suggestion
There are [faucets](https://en.bitcoin.it/wiki/Testnet#Faucets) like [this one](https://coinfaucet.eu/en/btc-testnet/) or [bitcoinfaucet.uo1.net](https://bitcoinfaucet.uo1.net/) that give a certain amount of testnet bitcoin per time period.
```

turbolay commented:
@yahiheb ready for merge
